service/account/providers/aws/account.go

```diff
@@ -0,0 +1,182 @@
+package aws
```

Comment:
this file should be kept outside aws folder, as this will be common for all aws, azure and gcp

Reply:
Fields in GCP and Azure can be different, so chose to keep it separate in aws directory.